### PR TITLE
Removing type -P and --force from script

### PR DESCRIPTION
**Pull request by hugowschneider**
This PR refers to the issues https://github.com/opendevstack/ods-core/issues/331, https://github.com/opendevstack/ods-core/issues/335 and https://github.com/opendevstack/ods-core/issues/336

In the script https://github.com/opendevstack/ods-core/blob/master/create-projects/create-cd-jenkins.sh:

- It removes the `type -P`
- Refactor the command line parser adding usage 
- Remove the --force from the script and adds it as a argument to the script
- Add in the Jenkins file --force to keep compatibility

\closes https://github.com/opendevstack/ods-core/issues/331
\closes https://github.com/opendevstack/ods-core/issues/335
\closes https://github.com/opendevstack/ods-core/issues/336

### CODE REVIEW

File: create-projects/create-cd-jenkins.sh [hunk 1]
```diff
@@ -1,80 +1,82 @@
...
+   printf "\t-h|--help\tPrints the usage\n"
+   printf "\t-d|--debug\tEnabled debug\n"
+   printf "\t-t|--tailor\tChanges the executable of tailor. Default: tailor\n"
+
+}
+
 
+while [[ "$#" -gt 0 ]]; do case $1 in
+   --status) STATUS=true;;
+   -d|--debug) DEBUG=true; set -x;;
```
Comment (michaelsauter) [inline create-projects/create-cd-jenkins.sh]: I like it, but that was the default previously. Maybe we add it to the arguments in the Jenkinsfile? Also, I would prefer this to be called `--verbose` to align with Tailor

File: create-projects/create-cd-jenkins.sh [hunk 2]
```diff
@@ -1,80 +1,82 @@
...
+   --force) FORCE="--force"; ;;
+   -h|--help) usage; exit 0;;
+   -t=*|--tailor=*) TAILOR="${1#*=}";;
+   -t|--tailor) TAILOR="$2"; shift;;
```
Comment (michaelsauter) [inline create-projects/create-cd-jenkins.sh]: Can we separate each group of args by a newline? Think that would make it even clearer ...

Comment (hugowschneider) [reply create-projects/create-cd-jenkins.sh]: Done :)

File: create-projects/create-cd-jenkins.sh (hunk 1 above)
Comment (hugowschneider) [reply create-projects/create-cd-jenkins.sh]: Done!

**Pull request merged**